account-admin/byob.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Bring Your Own S3 Bucket"
+description: ""
+---
+
+Users of the Enterprise plan can create their own S3 buckets and configure them to work with Rive. <br />
+[Contact us](https://rive.app/enterprise) to sign up for an Enterprise plan.
+
```

Comment:
I think we should also mention that customers will need to get in touch with Rive if they want to set up a custom bucket, as there's setup we'll need to do on our side. We sort of imply that towards the end with "Pass the following to your Rive representative", but we should make it explicit from the start.

We should also mention that if you have existing data in Rive you would like to have migrated to your S3 bucket, they should get in contact with us regarding that migration

Reply:
Also, we don't explain anywhere here "why" a customer would want a "custom bucket", or what it even is. I'm not sure if that is a separate doc or not